en/docs/how-tos/rate-limiting/distributed-throttling.md

```diff
@@ -1,14 +1,23 @@
 # Distributed Throttling
 
-WSO2 API Microgateway has an in-memory mechanism by default to handle throttling(node-level throttling).  In a distributed Microgateway deployement, throttling becomes a challenge with the current implementation. This is because, throttling decisions are taken by evaluating the local counter maintained within each node. Hence, scaling the microgateway will multiply the allowed limits. I.e. for e.g if the throttling limit is set to 10, if we have 3 gateways in a cluster, it will allow 30 requests to pass to the backend, before all three gateways throttle out requests. This will put an unexpected load on the backend. In order to address this, the API Microgateway supports distributed throttling where it is able to work with a central traffic management solution.
-
-The API Microgateway upon recieving a request, checks against the local counter and if throttling limit  has not exceeded it publishes the events via a stream to a central traffic management solution. This is done over HTTP. The  central traffic management solution then  executes throttle policies against the events streams. When a particular request is throttled, the  central traffic management solution sends the details of the throttled out event to a JMS topic. Each API Microgateway node is subscribed to this JMS topic, and updates the local counter when the JMS topic is updated.  Hence the API Microgateway nodes gets notified of the throttle decisions through JMS messages.
-
-![distributed throttling]({{base_path}}/assets/img/how-tos/distributed-throttling.png)
+WSO2 API Microgateway has an in-memory mechanism by default, to handle throttling
+([node-level throttling]({{base_path}}/how-tos/rate-limiting/adding-throttling-policies)).  
```

Comment:
I've noticed if we define urls in format `{{base_path}}/<path>` it always open the link in a new tab assuming its a external link. however if we omit `{{base_path}}` and keep only the absolute path to the linked page (from site root `/`) it opens in the same tab.
```suggestion
([node-level throttling](/how-tos/rate-limiting/adding-throttling-policies)).  
```